python/lsst/meas/algorithms/astrometrySourceSelector.py

```diff
@@ -0,0 +1,121 @@
+# See COPYRIGHT file at the top of the source tree.
```

Comment:
Word from SQuaRE is that we can't quite do the switch yet on existing projects. @jonathansick is planning a full switch for the whole codebase VerySoonNow™.